# HG changeset patch
# User Rik <rik@octave.org>
# Date 1349282474 25200
#      Wed Oct 03 09:41:14 2012 -0700
# Node ID afc002de3ade9673ce32b9d6918f7b0a261965ea
# Parent  d5c04a5c881752214743fe1b1e84296fe35528a3
Don't filter out Texinfo comments in built-in docstrings file.

* help.cc (install_built_in_docstrings): Don't filter out Texinfo comments in
built-in docstrings file.  Remove stray "octave_time t2" declaration.

diff --git a/libinterp/interpfcn/help.cc b/libinterp/interpfcn/help.cc
--- a/libinterp/interpfcn/help.cc
+++ b/libinterp/interpfcn/help.cc
@@ -981,37 +981,29 @@ install_built_in_docstrings (void)
 
           if (ov.is_defined ())
             {
               octave_function *fp = ov.function_value ();
 
               if (fp)
                 {
                   tmp = tmp.substr (pos+1);
-
-                  while (tmp.length () > 2 && tmp[0] == '@' && tmp[1] == 'c')
-                    {
-                      pos = tmp.find ('\n');
-                      tmp = tmp.substr (pos+1);
-                    }
-
                   fp->document (tmp);
                 }
             }
         }
     }
   else
     {
       // See note above about using std::cerr instead of warning.
 
       std::cerr << "warning: docstring file '" << fname << "' not found"
                 << std::endl;
     }
 
-  octave_time t2;
 }
 
 static void
 do_get_help_text (const std::string& name, std::string& text,
                   std::string& format)
 {
   bool symbol_found = false;
   text = raw_help (name, symbol_found);
